react/src/pages/analysis/AddAnalysisAlert.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { Button, ButtonGroup,  Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@material-ui/core';
+import { useHistory } from 'react-router';
+
+interface AddAnalysisAlertProps {
+    open: boolean,
+    onAccept: () => void,
+    onClose: () => void
+}
+
+export default function AddAnalysisAlert(props: AddAnalysisAlertProps) {
+    return (
+        <Dialog
+            open={props.open}
+            onClose={props.onClose}
+            aria-labelledby="alert-dialog-title"
+            aria-describedby="alert-dialog-description"
+        >
+            <DialogTitle id="alert-dialog-title">{"Add new Analysis"}</DialogTitle>
```

Comment:
Don't need curly braces around the body of the title. I think `Request analysis` might be a better title.